src/creature.h

```diff
@@ -186,6 +205,15 @@ class Creature
                                                    double total_dispersion );
 
         /**
+         * Makes an aiming/attack roll for a single projectile attack shot.
+         *
```

Comment:
Sorry about that. I guess it's just a case of getting used to it really. Consistency is important but the fact that you write documentation at all is welcome (compared to a lot of the legacy codebase which is entirely undocumented)